test/ably/resthttp_test.py

```diff
@@ -135,3 +137,12 @@ def test_custom_http_timeouts(self):
         self.assertEqual(ably.http.http_open_timeout, 8)
         self.assertEqual(ably.http.http_max_retry_count, 6)
         self.assertEqual(ably.http.http_max_retry_duration, 20)
+
+    def test_version_headers(self):
```

Comment:
Can we add a comment with `RSC7a` and `RSC7b` into the comments so that it's easy for others to search for spec items matching?